code/style.css

```diff
@@ -138,3 +150,21 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+/* Added styling to dropdown list */
+#select {
+  width: 100%;
+  cursor: pointer;
+  color: black;
+  font-family: 'Montserrat';
+  font-weight: 600;
+  padding: 16px 20px;
+  border-radius: 4px;
+  border: none;
+  font-size: 16px;
+  background: #e9e3e3;
+}
```

Comment:
You should avoid using id as CSS selector. It can raise a lot of issues in the bigger projects where you use many of the CSS files to style it. Remember that you can always create a class and name it the same as id. 🤓